EstFoisy/theia
# Scrollbar size settings in preference implemented

#### What it does
When the horizontal and vertical scrollbar sizes are adjusted in setting, it should reflect on the scrollbar height in the editor.

Closes #11732

#### How to test
1.	Open a new file 
2.	Move the cursor to the right until the bottom scrollbar appear 
3.	Move the cursor down until the side scrollbar appear 
4.	Then go into file: preference: Open Setting
5.	In the top tab bar type: scrollbar size
6.	Change the value of the horizontal and vertical scrollbar size.
7.	Go back into the file, move the cursor and the scrollbar should adjust according to the width and height of the preference.


https://user-images.githubusercontent.com/113064863/202552262-d1214fdc-75ab-4be6-9616-0934b3df7558.mp4


#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
